### PR TITLE
Create 2023_tech_tackle.md

### PR DESCRIPTION
**Pull request by akhil-maker**
Added a new haiku

### CODE REVIEW

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! There are some changes that need to be made to the syllable count in order to have a valid haiku. 

A haiku is composed of 3 lines. The first line is 5 syllables long, the second line is 7 syllables long, and the third line is 5 syllables long. Although the majority of the haikus here are written in English, you can submit a haiku in any human language. **Verify your haiku syllables with a checker like [Haiku Syllable Counter](https://www.haikusyllablecounter.com/)**.

Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: maintainers will only go through one round of review, first reviews that are not sufficiently addressed will be closed and marked invalid. Also note that after more than two days of inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by akhil-maker** [comment]
@ltagliaferri I have made the changes as you suggested, 5 syllables in 1st, 7 in second and 5 in third line

File: _haikus/2023_tech_tackle.md
Comment (ltagliaferri) [inline _haikus/2023_tech_tackle.md:7]:
```suggestion
Stuck while coding and <br>
```

File: _haikus/2023_tech_tackle.md
Comment (ltagliaferri) [inline _haikus/2023_tech_tackle.md:8]:
```suggestion
programming. Get your duck to<br>
```

File: _haikus/2023_tech_tackle.md
Comment (ltagliaferri) [inline _haikus/2023_tech_tackle.md:9]:
```suggestion
 discuss the problems<br>
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]

**Comment by akhil-maker** [comment]
@ltagliaferri Committed the suggestions made by you. Hope they are correct now.

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉 🦆

**Pull request merged**
